configuration/di-authentication-development/vpc/parameters.json

```diff
@@ -70,5 +70,13 @@
   {
     "ParameterKey": "SecretsManagerApiEnabled",
     "ParameterValue": "Yes"
+  },
+  {
+    "ParameterKey": "VpcPeeringConnectionId",
+    "ParameterValue": "pcx-0bd64e7650672b45e"
+  },
+  {
+    "ParameterKey": "VpcPeeringRequesterCidr",
+    "ParameterValue": "10.0.0.0/16"
```

Comment:
Is this deliberately 16?

Reply:
Yes, because I have already tested the change in dev. This is the final state (step 3 of 3).